src/utils/user/index.js

```diff
@@ -1,10 +0,0 @@
-/*
-
-	ard-eventhub
-	by SWR audio lab
-
-*/
-
-module.exports = {
-	resolvePermissions: require('./resolvePermissions'),
```

Comment:
`resolvePermissions.js` doesn't exist so I deleted the folder/file `/user/index.js`